schema/snapcraft.json

```diff
@@ -412,6 +412,14 @@
             "description": "the snap epoch, used to specify upgrade paths",
             "format": "epoch"
         },
+        "compression": {
+            "description": "compression to use during packing",
+            "type": "string",
+            "enum": [
+                "lzo",
+                "xz"
+            ]
```

Comment:
maybe set the default here?

Reply:
so it's clear to the user what the default is...

Reply:
I want "snap pack" to choose what the default is, and maybe that varies depending on snap contents